Lib/pathlib.py

```diff
@@ -444,8 +444,12 @@ def symlink(a, b, target_is_directory):
     utime = os.utime
 
     # Helper for resolve()
-    def readlink(self, path):
-        return os.readlink(path)
+    if hasattr(os, "readlink"):
+        def readlink(self, path):
+            return os.readlink(path)
```

Comment:
`readlink = os.readlink`

Reply:
actually, it will make the `readlink` from a method to a static method. In other words, it will change the API behavior

Reply:
How does that change API behaviour?

Reply:
from a method to a static method.

Although people can use it as usual, it changes the API implementation detail. I think it should be an API behavior-change.

Reply:
Can you show a repro case? I still can't see how this would break an API.

Reply:
it will not break the API, It will change the method type. I can't get a better word to describe it. :(

Reply:
I get that, but I don't see why it matters. Everything else in `_NormalAccessor` uses `os` functions directly without wrapping.

Reply:
it makes sense, I will update the code